.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: 'Create a report to help us improve Prism '
+title: "[BUG] "
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+Please detail how you caused the bug
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+* If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via  https://imgur.com/ or similar.
+
+**Server (please complete the following information):**
+  - Prism Version: X.X.X
+  - Server Type: [Spigot|Paper|CraftBukkit]
+  - Server Version: X.X.X
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5471038)